webapp/javascript/components/FileList.tsx

```diff
@@ -33,13 +34,13 @@ const getBodyRows = (
       cells: [
         {
           value: (
-            <>
-              {profile.name}
+            <div className={styles.profileName}>
+              <span title={profile.name}>{profile.name}</span>
               {isRowSelected && <CheckIcon className={styles.checkIcon} />}
-            </>
+            </div>
           ),
         },
-        { value: profile.updatedAt },
+        { value: format(parseISO(profile.updatedAt), 'yyyy-MM-dd') },
```

Comment:
@Rperry2174 Not sure I understand the reasoning behind this. What if I am uploading multiple files (via `adhoc` command for example) in a short interval (minutes) and I am basing off that info to navigate through my files.

Reply:
sounds good for time lets do this format:
![image](https://user-images.githubusercontent.com/23323466/193862753-38f72536-d113-46a0-a74f-5dd2d8c2bfe5.png)

Lets add one more column with the suffix:
`cpu`, `mutex`, `inuse_objects`, etc